sass/custom/_paulus.scss

```diff
@@ -114,15 +141,79 @@ $primary-color: #049cdb;
   .current-version {
     .release-date {
       white-space: nowrap;
+      font-weight: normal;
+      font-size: 14px;
     }
   }
 
+  .links {
+    margin-top: 12px;
+  }
+
   .recent-posts {
-    .blog-date {
-      white-space: nowrap;
+    margin-bottom: 28px;
+    .post {
+      .title {
+        font-size: 18px;
+        font-weight: bold;
+      }
+
+      a {
+        color: $text-color;
+        display: block;
+
+        &:hover {
+          color: $primary-color
+        }
+      }
+
+    }
+  }
+
+  .blog-date {
+    white-space: nowrap;
+  }
+
+  .side-by-side {
+    display: flex;
+    justify-content: space-between;
+
+    div {
+      width: 45%;
     }
   }
 
+  .vision {
+    margin-bottom: 24px;
+
+    h2 {
+      margin-bottom: 0;
+    }
+
+    h3 {
+      margin-top: .75em;
+    }
```

Comment:
Can we make the `font-weight` of headers less? Like 400 or something.

Reply:
Definitely down to do that. I do wish we had a font that had more settings for boldness. Aka a 500 and 600 difference.